salesagility/SuiteCRM
# Fix #9460 - Calculate field action won't manage properly formulas with decimal fields

Closes #9460

## Description
As described in the Issue, Calculate field workflow actions won't manage properly decimal field. If the decimal separator is different than a ".", the action won't keep the user-defined separator, giving as a result a number multiplied by 100.

This PR adds a section where it checks the type of the destination field and formats the number accordingly.

## Motivation and Context
The Calculate field should work as expected with any kind of field

## How To Test This
1. Change the default user decimal separator to something else than "."
2. Add two decimal field to any module
3. Create a workflow that saves one decimal field to another decimal field
4. Run the workflow and check that the result is correct.

## Types of changes

### Final checklist
